tests/ha/test_control_pod_restart.py

```diff
@@ -924,14 +929,17 @@ def test_iam_bkt_cruds_during_ctrl_pod_failover(self):
                            endpoint_url=S3_CFG["s3_url"])
         LOGGER.info("Step 1: Created IAM user successfully")
 
-        LOGGER.info("Step 2: Perform IAM user and bucket creation/deletion in background")
+        LOGGER.info("Create %s iam users for deletion", num_users)
+        users = self.mgnt_ops.create_account_users(nusers=num_users)
+        LOGGER.info("Step 2: Perform IAM user creation/deletion in background")
```

Comment:
If I understand right, this will start creation first and once done for 150 users will go for deletion. Is that ok with req? Or do we want to start creation of 150 and deletion of 150 a swell in parallel?

Reply:
No, this will go for create-delete one by one.
So create and delete both are in for loop, so both operations will happen one by one.